### PR TITLE
Computer vision

### PR DESCRIPTION
**Pull request by Alonso94**

### CODE REVIEW

File: fl_compvis/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ we have two ways to grab images from basler camera:
+ * Using python library: [pyPylon](https://github.com/basler/pypylon)
+ * Using ROS packages: [pylon_camera](https://github.com/magazino/pylon_camera)
+                     and [camera_control_messages](https://github.com/magazino/camera_control_msgs)
+ and in both methods we have to install the Camera Software Suite for Linux.
+ 
+ ## Working guide:
+ * Follow the instruction in [Interfacing Basler camera to ROS](https://www.baslerweb.com/fp-1527759326/media/downloads/documents/application_notes/Interfacing_Basler_Cameras_with_ROS.pdf).
+ * After the first time follow that steps:
+    1. Run rosnode in terminal#1:
```
Comment (l1va) [inline fl_compvis/README.md]: Better to have one launch file that started pylon_camera and fl_compvis ser.py at once. 
And maybe add that we have to run catkin_make to generate messages.

File: fl_compvis/scripts/ser.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    success, newbox = tracker.update(frame)
+    if success:
+	xe = (newbox[0]+newbox[2])/2.0;
+        ye = (newbox[1]+newbox[3])/2.0;
+        print(xe,ye)'''
+
+def callback(req):
+    bridge=CvBridge()
+    client = actionlib.SimpleActionClient("/pylon_camera_node/grab_images_raw", GrabImagesAction)
+    client.wait_for_server(rospy.Duration.from_sec(10.0))
```
Comment (l1va) [inline fl_compvis/scripts/ser.py]: 10 seconds? too a lot... why is it so long?

File: fl_compvis/README.md (hunk 1 above)
Comment (Alonso94) [reply fl_compvis/README.md]: Good idea, I will work on it.
the user will run catkin_make during the process of installing camera packages (steps in the pdf file) so i didn't mention that.

File: fl_compvis/scripts/ser.py (hunk 2 above)
Comment (Alonso94) [reply fl_compvis/scripts/ser.py]: In the practice we don't wait, but its to ensure receiving image.

**Pull request merged**